src/components/organisms/plan/plan.html

```diff
@@ -0,0 +1,54 @@
+<div class="orga-plan">
+  <div class="message">
+    <h2 class="title">ライフスタイルに合わせて選べる多彩なプラン。</h2>
+    <p class="explanation">
+      平日のみのプラン、仕事帰りをコアタイムとした夕方以降が中心のプラン、休日をメインとしたプランなど、<br />
+      多彩なプランの中からそれぞれのお客様のライフスタイルに合わせてお選びいただけます。<br />
+      ビジター料金でのご利用も可能ですので、月額会員登録の前に体験をしたいという方もぜひご利用ください。
+    </p>
+  </div>
+  <div class="plan-list">
+    <ul class="list-group plan">
+      <li class="list-group-item plan-name">デイタイムプラン</li>
+      <li class="list-group-item plan-price">
+        <div class="price">¥5,000</div>
+        <div class="plan-span">月額</div>
+      </li>
+      <li class="list-group-item plan-information1">入会金0円</li>
+      <li class="list-group-item plan-information1">平日：9時～17時</li>
+      <li class="list-group-item plan-information1">土日祝：利用ナシ</li>
+      <li class="list-group-item plan-information2">
+        コアタイムでの利用に最適なプラン
+      </li>
+    </ul>
+    <ul class="list-group plan">
+      <li class="list-group-item plan-name">スタンダードプラン</li>
+      <li class="list-group-item plan-price">
+        <div class="price">¥15,000</div>
+        <div class="plan-span">月額</div>
+      </li>
+      <li class="list-group-item plan-information1">入会金0円</li>
+      <li class="list-group-item plan-information1">平日：17時～23時</li>
+      <li class="list-group-item plan-information1">土日祝：営業時間全て</li>
+      <li class="list-group-item plan-information2">
+        営業時間内で終日ご利用可能なプラン
+      </li>
+    </ul>
+    <ul class="list-group plan">
+      <li class="list-group-item plan-name">プレミアムプラン</li>
+      <li class="list-group-item plan-price">
+        <div class="price">¥20,000</div>
+        <div class="plan-span">月額</div>
+      </li>
+      <li class="list-group-item plan-information1">入会金5000円</li>
+      <li class="list-group-item plan-information1">平日：営業時間全て</li>
+      <li class="list-group-item plan-information1">土日祝：営業時間全て</li>
+      <li class="list-group-item plan-information2">
+        フルに使いたい方のためのプ レミアムなプラン
+      </li>
+    </ul>
+  </div>
+  <div class="orga-button">
+    <a href="#" class="button">プラン料金</a>
+  </div>
```

Comment:
```suggestion
  <div class="button-wrapper">
    <a href="#" class="button">プラン料金</a>
  </div>
```
現状のクラス名だと organismsm/button というパーツがあるという誤った印象を与えかねないと思います。
このパーツの目的はボタンを中央寄せすることと余白の管理だと思うので、上記のようなクラス名の方が適切な気がします！